app/src/online/java/com/kamron/pogoiv/updater/AppUpdateUtil.java

```diff
@@ -94,4 +103,15 @@ public void onClick(DialogInterface dialogInterface, int i) {
         AlertDialog dialog = builder.create();
         return dialog;
     }
+
+    public static void deletePreviousApkFile(Context context) {
+        if (ContextCompat.checkSelfPermission(context,
+                Manifest.permission.WRITE_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED){
+            String newApkFilePath = context.getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS) + "/" + FILE_NAME;
```

Comment:
Going by vague memory of discussion about the app's size bloat, I thought the downloaded apk was stored in GoIV's private data section.  Is it not? Shouldn't that be the one that's deleted? Also, now that the app is on the play store, do we even need this autoupdater?